--- client.c.orig	2026-09-18 11:51:08
+++ client.c	2026-09-18 11:51:08
@@ -25,7 +25,7 @@
 	if (has_term)
 		tcsetattr(STDIN_FILENO, TCSAFLUSH, &orig_term);
 	if (alternate_buffer) {
-		printf("\033[?25h\033[?1049l");
+		printf("\033[?25h");
 		fflush(stdout);
 		alternate_buffer = false;
 	}
@@ -46,7 +46,7 @@
 	tcsetattr(STDIN_FILENO, TCSANOW, &cur_term);
 
 	if (!alternate_buffer) {
-		printf("\033[?1049h\033[H");
+		/* cove: stay on the main screen so kitty keeps scrollback */
 		fflush(stdout);
 		alternate_buffer = true;
 	}
